server/bundler/babel/babel-lodash-es/index.js

```diff
@@ -0,0 +1,54 @@
+/** @format */
+
+const types = require( 'babel-types' );
+
+module.exports = function() {
+	return {
+		visitor: {
+			ImportDeclaration( path ) {
+				const { source } = path.node;
+
+				// Transform any import from 'lodash-es' (exact match, no submodule) to 'lodash'
+				const fullMatch = source.value.match( /^lodash-es$/ );
+				if ( fullMatch ) {
+					source.value = 'lodash';
+					return;
+				}
+
+				// Transform default import from a 'lodash-es' submodule to a a named import from 'lodash'
+				// Example:
+				// In: import theGet from 'lodash-es/get'
+				// Out: import { get as theGet } from 'lodash'
+				const subMatch = source.value.match( /^lodash-es\/(.*)/ );
+				if ( subMatch ) {
+					const { specifiers } = path.node;
+					// If there is anything else than a single default import, throw an error. Such an
+					// import is not valid. Example: import { get } from 'lodash-es/get'
+					// There is only the default export in 'lodash-es/get' and no named one.
+					if (
+						! specifiers ||
+						specifiers.length !== 1 ||
+						specifiers[ 0 ].type !== 'ImportDefaultSpecifier'
+					) {
+						throw path.buildCodeFrameError(
```

Comment:
never seen a `buildCodeFrameError`. very cool

Reply:
I found it when looking at transforms in core Babel, trying to learn how they report errors.